compiler/src/dotty/tools/dotc/printing/SyntaxHighlighting.scala

```diff
@@ -284,9 +285,20 @@ object SyntaxHighlighting {
         case ':' => true
         case '@' => true
         case '.' => true
+        case ',' => true
+        case ')' => true
         case _ => false
       }
 
+      def shouldUpdateLastToken(currentPotentialToken: String): Boolean =
```

Comment:
Could you add a documentation comment for this method?

Reply:
Yes, I will write documentation to this.
Also I'm thinking about renaming currentPotentialToken to just currentToken. "Potential" wanted to express that it might be the next "lastToken", but now I don't think it adds any value to the name.